benchmarks/loops-times.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../harness/loader'
+
+# Fix these values for determinism
+u = 5
+r = 7
+
+run_benchmark(10) do
+  a = Array.new(10000, 0)
+
+  4_000.times do |i|
+    4_000.times do |j|
+      a[i] += j % u
```

Comment:
This modulo in the inner loop might likely dominates the benchmark, I'd suggest to use `& 0b111` or so instead.